src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -286,8 +286,8 @@ RSC_012=Fragment identifier is not defined.
 RSC_013=Fragment identifier is used in a reference to a stylesheet resource.
 RSC_014=Fragment identifier defines an incompatible resource type.
 RSC_015=A fragment identifier is required for svg use tag references.
-RSC_016=Fatal Error while parsing file '%1$s'.
-RSC_017=Warning while parsing file '%1$s'.
+RSC_016=Fatal Error while parsing file: %1$s
+RSC_017=Warning while parsing file: %1$s
```

Comment:
Then I'd say we stick to `X while parsing file: {message}`. @rdeltour okay for you?